ports/openfx/Openfx.cmake

```
@@ -0,0 +1,56 @@
+cmake_minimum_required(VERSION 3.20)
+
+project(openfx VERSION 1.4.0 LANGUAGES CXX)
+
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+if(WIN32)
+    add_compile_definitions(WINDOWS)
+    add_compile_options(/DNOMINMAX)
+    add_definitions(-DWIN64)
+    set(OS_VAR "windows")
+    set(OFX_ARCH_NAME "Win64")
+endif()
+
+set(OFX_HEADERS_DIR ${CMAKE_CURRENT_SOURCE_DIR}/include CACHE INTERNAL "OFX_HEADERS_DIR")
```

Internal cache variables are forced, not input. Usually pointless at top level.
```suggestion
set(OFX_HEADERS_DIR "${CMAKE_CURRENT_SOURCE_DIR}/include")
```